.changeset/rich-singers-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': major
```

Comment:
Oh oh 😅 

Can we support both until the next major so that this is not a breaking change.

<small>because we just shipped this, i'm also okay with sneaking it in if no one has used it yet</small>

Reply:
I confirmed that this is not being used anywhere yet. Let's sneak it in 😈

Reply:
Also, consumer tests are failing. Is there anything I am missing? Doesn't seem coming from this PR's changes though.